clvm/more_ops.py

```diff
@@ -68,6 +68,15 @@ def args_as_ints(op_name, args):
         yield (arg.as_int(), len(arg.as_atom()))
 
 
+def args_as_int32(op_name, args):
```

Comment:
What do you think about moving this guy to `casts.py`?

Reply:
Oh I guess that doesn't make sense, nm.

Reply:
`args_as_int32` could take an arg_count param

Reply:
note that this is actually a generator, so it would be a complicated place to check number of arguments. However, this is only used after the number of arguments are already checked